lib/developer_portal/app/controllers/developer_portal/admin/account/payment_details_base_controller.rb

```diff
@@ -28,7 +28,7 @@ def edit
   def update
     current_account.updating_payment_detail = true
     if current_account.update account_params
-      redirect_to payment_details_path, notice: 'Your billing address was successfully stored'
+      update_billing_address_on_stripe(account_params)
```

Comment:
This line is a bit concerning, because this controller is generic, and not Stripe-specific.
I've been looking in to this, and it seems that we currently support just Stripe and Braintree, and Braintree doesn't use this path, so it's kind of fine (in the sense that it doesn't break the Braintree flow). Also, it seems that we can't move this to the Stripe controller, because we don't want to break the existing portals.

But I think we do need to at least check here whether the current account's payment gateway is actually stripe, before trying to update anything.

Reply:
Hm... probably we can actually change this (e.g. by moving the method to the Stripe Controller) without breaking the portals, because this path is not in the template, but in the Liquid drop: https://github.com/3scale/porta/blob/0155b4a05248c6915086bbd13fd2b4305671e71c/lib/developer_portal/lib/liquid/forms/account.rb#L11

But we need to think about whether that's the right thing to do.